app/models/contact.rb

```diff
@@ -5,6 +5,8 @@ class Contact < ApplicationRecord
   validates :contact_type, presence: true
   validate :validate_emails
 
+  attr_readonly :contact_type_id
```

Comment:
Is this needed?

Reply:
I understand why you've done this from looking at the test. The better way to prevent updating an attribute is via StrongParameters in the controller. You can have a different set of allowed params for create and update.